settings/base.py

```diff
@@ -32,14 +32,10 @@
 # See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'SECRET_KEY'
 
 SECRET_KEY = config('SECRET_KEY')
-DATABASES = {
-    'default': dj_database_url.config(
-        default=config('DATABASE_URL')
-    )
-}
+
+DATABASES = {'default': dj_database_url.config()}
```

Comment:
Will this allow flexibility to allow each dev to use their own environment variables ?

Reply:
LOL! I've merged! But yes @HawiCaesar it should.

Reply:
@HawiCaesar should be fine as long as you have `DATABASE_URL` env variable defined